R/misc_yhat.R

```diff
@@ -73,6 +73,11 @@ yhat.svm <- function(X.model, newdata, ...) {
 yhat.gbm <- function(X.model, newdata, ...) {
   n.trees <- X.model$n.trees
   response <- predict(X.model, newdata = newdata, n.trees = n.trees, type = "response")
+  #gbm returns and 3D array for multilabel classif
+  if(!is.null(dim(response)[3])){
```

Comment:
Let's not use numerical indexes if not needed. 
In this `If` you can check if `dim()` is shorter than 2 elements `length(dim(...))`

Reply:
Done